metadata/4506T__1__metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "4506T",
+    "retrainable": true,
+    "gpu": 0,
+    "processorType": "CPU",
+    "description": "Machine Learning model(available in Preview) for extracting commonly occurring data points from 4506T documents, including header fields and line items. Please see more details including supported languages and link to Activities guide in the About Licensing -> Document Understanding API Key section of the UiPath Automation Cloud Guide here: https://docs.uipath.com/automation-cloud/docs/about-licensing#section-document-understanding-api-key",
+    "inputDescription": "ML Skills deployed using this ML Package can be integrated into RPA workflows using the Machine Learning Extractor activity in the Official feed or the MLModel activity in Connect feed. File formats accepted include pdf, tiff, jpg or png files. The activities also require an API Key input which you need to obtain from your UiPath Automation Cloud account, in the Licenses -\u003e Other Services view.",
+    "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+    "changeLog": "Model: 22.4.0",
+    "cpu": 1,
+    "inputType": "JSON",
+    "displayName": "4506T",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON37_DU",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Document Understanding",
+    "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v21.10",
```

Comment:
@vivekjhauipath @vnaren23 Which is the min ai fabric version that work with language version 4?

Reply:
Language version 4 will only go with AIC 22.4. Also version will be `22.4` and not ~~`v22.4`~~

Reply:
Done